lib/vector-core/src/config/log_schema.rs

```diff
@@ -84,8 +83,8 @@ impl Default for LogSchema {
 }
 
 impl LogSchema {
-    fn default_message_key() -> String {
-        String::from("message")
+    fn default_message_key() -> OptionalValuePath {
+        OptionalValuePath::new("message")
```

Comment:
Should these be in a `const` elsewhere?

Reply:
Are you referring to the hardcoded string or `OptionalValuePath`? Happy to refactor this (and the other default_* functions).

Reply:
The hard coded string constant. We just added a style doc note about using `const` values for those.